internal/commands/secrets/delete_test.go

```diff
@@ -0,0 +1,310 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+)
+
+func TestSecretHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	app := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "eggcorn-abcde",
+		Name:        "eggcorn",
+	}
+
+	testLen := 7
+	secrets := make([]realm.Secret, testLen)
+	for i := 0; i < testLen; i++ {
+		secrets[i] = realm.Secret{
+			ID:   fmt.Sprintf("secret_id_%d", i),
+			Name: fmt.Sprintf("secret_name_%d", i),
+		}
+	}
+	for _, tc := range []struct {
+		description    string
+		testInput      []string
+		expectedOutput []secretOutput
+		expectedErr    error
+	}{
+		{
+			description: "should return successful outputs for proper secret inputs",
+			testInput:   []string{"secret_id_1"},
+			expectedOutput: []secretOutput{
+				{
```

Comment:
[stylenit] these are small enough to be defined inline: `{secrets[1], nil}` (and this particular `[]secretOutput` array can probably fit all in-line